src/PIL/Jpeg2KImagePlugin.py

```diff
@@ -217,11 +214,24 @@ def load(self):
         if self.tile:
             # Update the reduce and layers settings
             t = self.tile[0]
-            t3 = (t[3][0], reduce, self.layers, t[3][3], t[3][4])
+            t3 = (t[3][0], self._reduce, self.layers, t[3][3], t[3][4])
             self.tile = [(t[0], (0, 0) + self.size, t[2], t3)]
 
         return ImageFile.ImageFile.load(self)
 
+    @property
+    def reduce(self):
+        return super().reduce
+
+    @reduce.setter
+    def reduce(self, value):
+        warnings.warn(
+            "The reduce property for Jpeg2KImageFile is deprecated "
+            "and will be removed in the next major version.",
```

Comment:
```suggestion
            "and will be removed in a future major version.",
```

Docs, emphasis added:

> Deprecated features are only removed in major releases **after an appropriate period of deprecation has passed**.

https://pillow.readthedocs.io/en/stable/deprecations.html#removed-features

Python 3.5 EOL is due in September, so Pillow 8 may be out in October (assuming a major bump for an EOL version drop).

We've gone for around a year for an "appropriate period" (https://github.com/python-pillow/Pillow/issues/1087#issuecomment-121655443) so let's keep the comment less specific.

Reply:
Which version this will be deprecated in? 7.0.1 or 7.0?

Reply:
7.1.0 is the next one due out, in April.

Reply:
ok, updated deprecations.rst